src/components/index.d.ts

```diff
@@ -286,6 +286,50 @@ export declare namespace RadioButtonGroup {
   }
 }
 
+export declare const Grid: {
+  (props: any): JSX.Element
+  propTypes: {
+    children: any
+    className?: any
+    rowRefs: any
+    columnRefs: any
+  }
+  defaultProps: {
+    className: any
+  }
+  displayName: string
+}
+export declare const Column: any
+export declare const Row: any
+export declare const Pagination: {
+  ({
+    fetchPageCallback,
+    renderCallback,
+  }: {
+    fetchPageCallback: any
+    renderCallback: any
+  }): JSX.Element
+  propTypes: {
+    fetchPageCallback: any
+    renderCallback: any
+  }
+  displayName: string
+}
+export declare const useGridSorting: (
+  rows: any,
```

Comment:
just for my understanding, do these really need to be `any`?

Reply:
Probably not but I plan to be iterative and refine these types further